x-pack/test/functional/apps/canvas/feature_controls/canvas_security.ts

```diff
@@ -66,7 +66,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
       it('shows canvas navlink', async () => {
         const navLinks = (await appsMenu.readLinks()).map((link) => link.text);
-        expect(navLinks).to.eql(['Canvas', 'Stack Management']);
```

Comment:
What is the change to this PR that was causing this to fail?

Reply:
I added a new item in the menu called `Overview` inside the Observability section, and navLinks now has three elements ['Overview', 'Canvas', 'Stack Management'], and since the test is called  `shows canvas navlink` I don't see any reason why it also checks for `Stack Management`.

Reply:
Gotcha! 👍